Doctrine/Orm/Filter/SearchFilter.php

```diff
@@ -50,6 +50,16 @@ class SearchFilter extends AbstractFilter
     const STRATEGY_WORD_START = 'word_start';
 
     /**
+     * @var string Doctrine expression to case insensitivity comparison
+     */
+    const CASE_INSENSITIVE = 'LOWER';
+
+    /**
+     * @var bool whether it's case sensitive or not
+     */
+    private $caseSensitive = true;
```

Comment:
A filter is not created for a given request and could be used many times for many requests, so I'm not fond of the idea of having this as a property of that class.